src/Etterna/Singletons/DownloadManager.cpp

```diff
@@ -2140,8 +2139,7 @@ DownloadManager::StartSession(
 	auto done = [user, pass, callback](HTTPRequest& req, CURLMsg*) {
 		Document d;
 		if (d.Parse(req.result.c_str()).HasParseError()) {
-			LOG->Trace(("Malformed request response: " + req.result).c_str());
-			DLMAN->loggingIn = false;
```

Comment:
Possible behavior change?

Reply:
removes behavior from bugfix da161866f6d4169b119f1001230d7e6ebad1b403

Reply:
Accidentally removed in rebase. Added again.